Podcast/FeedElementTableViewCell.swift

```diff
@@ -10,123 +10,64 @@ import UIKit
-            (feedElementSupplierView as! UserSeriesSupplierView).setupWithSeries(series: series)
-            (feedElementSupplierView as! UserSeriesSupplierView).delegate = self
-            feedElementSubjectView = EpisodeSubjectView(episode: episode)
-            (feedElementSubjectView as! EpisodeSubjectView).delegate = self
-            heightConstraint = newlyReleasedEpisodeSupplierViewHeight
-        }
+extension FeedElementTableViewCell where Self: UITableViewCell {
+    func initialize() {
+        selectionStyle = .none
 
-        contentView.addSubview(feedElementSubjectView)
-        contentView.addSubview(feedElementSupplierView)
+        contentView.addSubview(supplierView)
+        contentView.addSubview(subjectView)
 
-        feedElementSupplierView.snp.makeConstraints { make in
+        supplierView.snp.makeConstraints { make in
             make.leading.equalToSuperview()
             make.trailing.equalToSuperview()
             make.top.equalToSuperview()
-            make.height.equalTo(heightConstraint)
+            make.height.equalTo(supplierViewHeight) // should make this generic
         }
 
-        feedElementSubjectView.snp.makeConstraints { make in
+        subjectView.snp.makeConstraints { make in
             make.leading.equalToSuperview()
             make.trailing.equalToSuperview()
-            make.top.equalTo(feedElementSupplierView.snp.bottom)
+            make.top.equalTo(supplierView.snp.bottom)
             make.bottom.equalToSuperview()
         }
     }
-    
-    required init?(coder aDecoder: NSCoder) {
-        fatalError("init(coder:) has not been implemented")
-    }
-    
-    func supplierViewDidPressFeedControlButton(supplierView: UserSeriesSupplierView) {
-        delegate?.feedElementTableViewCellDidPressSupplierViewFeedControlButton(feedElementTableViewCell: self, supplierView: supplierView)
-    }
-    
-    func episodeSubjectViewDidPressPlayPauseButton(episodeSubjectView: EpisodeSubjectView) {
-        delegate?.feedElementTableViewCellDidPressEpisodeSubjectViewPlayPauseButton(feedElementTableViewCell: self, episodeSubjectView: episodeSubjectView)
-    }
-    
-    func episodeSubjectViewDidPressRecommendButton(episodeSubjectView: EpisodeSubjectView) {
-        delegate?.feedElementTableViewCellDidPressEpisodeSubjectViewRecommendedButton(feedElementTableViewCell: self, episodeSubjectView: episodeSubjectView)
-    }
-    
-    func episodeSubjectViewDidPressBookmarkButton(episodeSubjectView: EpisodeSubjectView) {
-        delegate?.feedElementTableViewCellDidPressEpisodeSubjectViewBookmarkButton(feedElementTableViewCell: self, episodeSubjectView: episodeSubjectView)
-    }
-    
-    func episodeSubjectViewDidPressTagButton(episodeSubjectView: EpisodeSubjectView, index: Int) {
-        delegate?.feedElementTableViewCellDidPressEpisodeSubjectViewTagButton(feedElementTableViewCell: self, episodeSubjectView: episodeSubjectView, index: index)
+}
+
+extension UITableView {
```

Comment:
can we have multiple UITableView extensions floating around? If not, we might want to abstract this out to a UITableView+ file just so all our extensions are in the same place

Reply:
Personally I like having them in relevant places since potentially we could end up with a huge file of extensions with no organization of where each one is or what each one does.
But honestly I don't mind. If you want to you can take it out when we get more extensions later.

Reply:
It's fine let's keep it in here for rn.